### ISSUE TITLE
Is it possible to output to a specific resolution/size?

### ISSUE THREAD

**Issue by carterbryden** [zhanziyang/vue-croppa#100]
I'm planning to use this in part to speed up image uploads to the server. 

Regardless of the size of image the user puts into croppa, I'd like to output say 1920px by 1080px, or any other size. Then I can send that blob or data url to the server, saving a ton of bandwidth if they are inputting huge pictures.

Is there a way to do this with croppa?

**Comment by zhanziyang** [comment]
Yes. See the "Output Image Dimension" panel in this page: https://zhanziyang.github.io/vue-croppa/#/customization

**Comment by zhanziyang** [comment]
@carterbryden Problem solved?

**Comment by carterbryden** [comment]
Thank you, yes it does.

One more question: 

If I put in say a picture thats 3000px wide and the croppa preview is 500px wide, can it output a genuine 1000px image? Or would it be a 500px image stretched to 1000px and distorted?

**Comment by zhanziyang** [comment]
@carterbryden  It's genuine 1000px image.

**Comment by zhanziyang** [comment]
Keeping this open as a feature request for vue-croppa 2.x.

Arbitrary output width and height independent of viewport aspect ratio requires a dedicated multi-size export API; currently `quality` scales the canvas maintaining the viewport ratio.
